contextualized/regression/lightning_modules.py

```diff
@@ -148,7 +148,7 @@ def _y_reshape(self, preds, dataloader):
         return ys
 
     def dataloader(self, C, X, Y, **kwargs):
-        return self._dataloader(C, X, Y, MultivariateDataset, **kwargs)
+        return self._dataloader(C, X, Y, MultivariateDataset, worker_init_fn=worker_init_fn, **kwargs)
```

Comment:
When is `worker_init_fn` meant to be changed from the default function?

It's a little strange to me to see `worker_init_fn=worker_init_fn`. It seems like either the same `worker_init_fn` is always intended to be used (in which case we don't need to pass it as an argument), or the provided `worker_init_fn` should be renamed `default_worker_init_fn`.

Reply:
Eventually we'll have two worker init functions as well (one for the single task metamodels and one for the multi task metamodels). That could also help with naming.